app/_hub/kong-inc/ai-semantic-cache/overview/_index.md

```diff
@@ -0,0 +1,156 @@
+    end
+
+    subgraph AISemanticCache["AI Semantic Cache"]
+        Plugin[AI Semantic Cache Plugin]
+    end
+
+    subgraph DataStorageRedis["Data Storage (Redis)"]
+        VectorDB[Vector Database Role]
+        Cache[Cache Role]
+    end
+
+    User -->|Request| Kong
+    Kong -->|Forward Request| Plugin
+    Plugin -->|Check Cache| Cache
+    Cache -->|Cache Hit| Plugin
+    Cache -->|Cache Miss| VectorDB
+    VectorDB -->|Retrieve Embedding| Plugin
+    Plugin -->|Generate Response| Kong
+    Kong -->|Send Response| User
+{% endmermaid %}
+
+### Vector databases
+
+A vector database can be used to store vector embeddings, or numerical representations, of data items. For example, a response would be converted to a numerical representation and stored in the vector database so that it can compare new requests against the stored vectors to find relevant cached items.
+
+Currently, the AI Semantic Cache plugin supports the following vector databases:
+
+* Redis (beta)
+* Qdrant (alpha)
+* PGVector (alpha)
+* Supabase (alpha)
+
+For more information, see [Deploy and manage a vector database](/hub/kong-inc/ai-semantic-cache/vector-database/).
+
+### Cache management
+
+With the AI Semantic Cache plugin, you can configure a cache of your choice to store the responses from the LLM.
+
+For more information, see [Cache management](/hub/kong-inc/ai-semantic-cache/cache-management/).
+
+#### Caching mechanisms
+
+The AI Semantic Cache plugin improves how AI systems provide responses by using two kinds of caching mechanisms:
+
+* **Exact Caching:** This stores precise, unaltered responses for specific queries. If a user asks the same question multiple times, the system can quickly retrieve the pre-stored response rather than generating it again each time. This speeds up response times and reduces computational load.
+* **Semantic Caching:** This approach is more flexible and involves storing responses based on the meaning or intent behind the queries. Instead of relying on exact matches, the system can understand and reuse information that is conceptually similar. For instance, if a user asks about "Italian restaurants in New York City" and later about "New York City Italian cuisine," semantic caching can help provide relevant information based on their related meanings.
+
+Together, these caching methods enhance the efficiency and relevance of AI responses, making interactions faster and more contextually accurate.
+
+### Configuration parameters
+
+You can configure the following required parameters for the AI Semantic Cache plugin:
+
+* `vectordb`: Specifies the vector database strategy and settings
+* `embeddings`: Defines the embeddings provider, model, and configuration
+* `cache_control`: Enables or disables Cache-Control header handling
+* `storage_ttl`: Sets the time-to-live for cached responses
+* `exact_caching`: Enables or disables exact caching
+* `message_countback`: Specifies how many messages to consider for context
```

Comment:
@AntoineJac Is this how many messages the plugin will go through to find a semantic match before determining that there is no match?